src/App/app.jsx

```diff
@@ -28,25 +29,28 @@ const App = ({ root_store }) => {
             <MobxProvider store={root_store}>
                 {
                     root_store.ui.is_mobile || (root_store.ui.is_tablet && isTouchDevice) ?
-                        <Wip /> :
+                        <React.Suspense fallback={<UILoader />}>
+                            <Wip />
+                        </React.Suspense> :
                         <React.Fragment>
                             <Header />
+                            <React.Suspense fallback={<UILoader />} >
```

Comment:
@easteregg IMO, it does not need to show any loader for this component.

Reply:
We can pass an empty `div` instead `<React.Suspense fallback={<div />} >` as a fallback.

Reply:
Yeah, or don't pass any fallback at all. Absolutely.

Reply:
Turns out it spits an error if we don't declare the fallback :joy: I'll keep this shameful comment here, to remind myself never speak without reason.